samples/crypto/persistent_key_usage/sample.yaml

```diff
@@ -1,20 +1,23 @@
 sample:
-    description: This app provides an example of using peristent keys with the
-        PSA APIs. A random AES persistent key is generated and imported
-        to the PSA keystore.
-    name: Persistent key example
+  description:
```

Comment:
Is the `|` missing here? Or it's intentional ?

Reply:
Oh sorry. This one seems to be missing

Reply:
Fixed. Rerunning CI